tests/test_browser.py

```diff
@@ -3924,11 +3924,9 @@ def test_pthread_condition_variable(self):
   # Test that pthreads are able to do printf.
   @requires_threads
   def test_pthread_printf(self):
-    def run(debug):
-       self.btest(test_file('pthread/test_pthread_printf.cpp'), expected='0', args=['-s', 'INITIAL_MEMORY=64MB', '-O3', '-s', 'USE_PTHREADS', '-s', 'PTHREAD_POOL_SIZE', '-s', 'LIBRARY_DEBUG=%d' % debug])
-
-    run(debug=True)
-    run(debug=False)
+    args = ['-s', 'INITIAL_MEMORY=64MB', '-O3', '-s', 'USE_PTHREADS', '-s', 'PTHREAD_POOL_SIZE']
+    # args += ['-sLIBRARY_DEBUG']
```

Comment:
I think this was somewhat useful, as I think we added it when that option broke. But I agree it's not worth keeping around all the time. How about removing the line, though, or adding a comment?
```suggestion
    # It may be useful to also test LIBRARY_DEBUG here:  args += ['-sLIBRARY_DEBUG']
```